src/dali/plugin/input/csv/blockfi.py

```diff
@@ -111,7 +137,25 @@ def load(self) -> List[AbstractTransaction]:
                             crypto_received=line[self.__AMOUNT_INDEX],
                         )
                     )
-                elif line[self.__TYPE_INDEX] == _WITHDRAWAL:
+                elif entry_type == _ACH_WITHDRAWAL:
```

Comment:
Are crypto sales marked as "Ach Withdrawal" in BlockFi? I was confused by this because ACH brings to mind a movement of fiat funds, but, based on this code, it looks like they mean sale of crypto (into fiat). I didn't do any trades on BlockFi myself, so don't have a trade CSV file to look at.

Reply:
USD (fiat) redrawals in blockfi are tracked as GUSD tokens

Reply:
Ok, then this is fine. GUSD has slight price variations compared to USD and so it can trigger taxable events, I guess, so it's ok to track it. I'm asking these questions because my BlockFi activity is fairly simple and I didn't see the transaction types you introduced with this PR.

Reply:
yeah, unfortunately it results in unnecessary gains. blockfi also sends out 1099 to IRS with each transaction tracked in GUSD, so there is no easy way to work around it.